plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnector.java

```diff
@@ -219,12 +219,15 @@ public Version getVersion() throws IOException {
      * @throws InterruptedException
      *         if build process was interrupted
      */
-    public String buildImage(String repository, ProgressMonitor progressMonitor, AuthConfigs authConfigs, File... files)
-            throws IOException, InterruptedException {
+    public String buildImage(String repository,
+                             ProgressMonitor progressMonitor,
+                             AuthConfigs authConfigs,
+                             boolean doForcePull,
```

Comment:
After discussions we agreed to leave this PR with current style of parameters and create an issue for addition config objects to all methods of docker client. Is it OK for you guys?